Fixes:
Traceback (most recent call last):
  File "launchmany-curses.py", line 319, in <module>
    platform.decode_from_filesystem(torrent_dir)
AttributeError: 'module' object has no attribute 'decode_from_filesystem'
diff -Naur BitTorrent-4.26.0.orig/launchmany-curses.py BitTorrent-4.26.0/launchmany-curses.py
--- BitTorrent-4.26.0.orig/launchmany-curses.py	2006-10-11 00:25:35.000000000 +0200
+++ BitTorrent-4.26.0/launchmany-curses.py	2006-10-22 23:40:34.000000000 +0200
@@ -316,7 +316,7 @@
         if args:
             torrent_dir = args[0]
             config['torrent_dir'] = \
-                platform.decode_from_filesystem(torrent_dir)    
+                decode_from_filesystem(torrent_dir)    
         else:
             torrent_dir = config['torrent_dir']
             torrent_dir,bad = platform.encode_for_filesystem(torrent_dir)
